sdk/servicebus/azure-servicebus/azure/servicebus/_common/constants.py

```diff
@@ -80,6 +75,18 @@
 MESSAGE_DEFER = 'defer'
 MESSAGE_RENEW_LOCK = 'renew'
 
+SETTLEMENT_COMPLETE = "completed"
+SETTLEMENT_ABANDON = "abandoned"
+SETTLEMENT_DEFER = "defered"
+SETTLEMENT_DEADLETTER = "suspended"
+
+MESSAGE_MGMT_SETTLEMENT_TERM_MAP = {
```

Comment:
Really really dumb maintainability nit: If I'm reading this as an uninitiated, I wouldn't know which side maps to which. (mgmt vs non mgmt) May be worth a comment giving a tiny bit of clarity.

Reply:
added one line of note